package/scripts/deno_std/download.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
-export DENO_DIR=src/resources/deno_std/cache 
-package/dist/bin/tools/deno cache --unstable --lock src/resources/deno_std/deno_std.lock "$@" package/scripts/deno_std/deno_std.ts
-
+if [ -z "$QUARTO_DENO" ]; then
```

Comment:
Currently we run the lock script (which alongside this) to periodically lock the embedded version of the deno std library. This script calls download.sh so currently will fail if run from the top level.